docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -8,6 +8,29 @@
 - [v2.26.5](#v2265)
 - [v2.26.6](#v2266)
 - [v2.26.7](#v2267)
+- [v2.26.8](#v2268)
+
+## v2.26.8
+
+**GitHub release: [v2.26.8](https://github.com/kubermatic/kubermatic/releases/tag/v2.26.8)**
+
+### Supported Kubernetes versions
+
+- Add 1.31.8/1.30.12 to the list of supported Kubernetes releases ([#14395](https://github.com/kubermatic/kubermatic/pull/14395))
+
+### Bugfixes
+
+- Fix a bug for KubeLB where disabling the ingress class for a user cluster was not working ([#14396](https://github.com/kubermatic/kubermatic/pull/14396))
+- Add role prioritization: Update logic to return the highest-priority role for members with multiple roles ([#7272](https://github.com/kubermatic/dashboard/pull/7272))
+- Add special characters restriction on Inputs and escape values to avoid rendering as HTML ([#7288](https://github.com/kubermatic/dashboard/pull/7288))
+
+### Updates
+
+- Security: Update Cilium to 1.15.16 because the previous version is affected by CVE-2025-32793 ([#14435](https://github.com/kubermatic/kubermatic/pull/14435))
+- Update cert-manager to v1.16.5Action Required: In the cert-manager values.yaml, following updates should be done  - update  `webhook.replicas` to `webhook.replicaCount` - update  `cainjector.replicas` to `webhook.replicaCount` - remove `webhook.injectAPIServerCA` ([#14400](https://github.com/kubermatic/kubermatic/pull/14400))
```

Comment:
SAme formatting errors as in the 2.26 PR

Reply:
my bad, fixed!